DESIGN_PRINCIPLES.md

```diff
@@ -0,0 +1,24 @@
+# Concourse Design Principles
+
+Concourse's goal is to solve automation, once and for all, without becoming part of the problem.
+
+
+## Expressive by being precise
+
+Concourse should provide concepts that build a strong mental model for the user's project that remains intuitive as their automation requirements grow.
```

Comment:
Suggestion:
"Concourse should provide concepts that build a strong mental model for the user's project, **and this model should remain** intuitive as **the user's** automation requirements grow.